### DISCUSSION TITLE
How to force newlines to not be surrounded by <p> tag?

### DISCUSSION THREAD

**Discussion by douglasrcjames** [tinymce/tinymce#8987] [General]
Wondering if there is a way to easily force the newlines (or `<br>` tags) to not be surround by <p> tags. The use case is I have a client copy pasting content from a document into the Tiny MCE, but the newlines are being converted into empty <p> tags which gives the appearance of an extra large newline, which isnt ideal. Looks like setting `forced_root_block` in init to just a space will fix this issue, but obviously create new ones as warned in the [documentation](https://www.tiny.cloud/docs/tinymce/6/content-filtering/#forced_root_block), so that is not the fix for this. It would be great if `forced_root_block` could selectively apply to specific tags, excluding newlines/`<br>`.

Anyone have a work around for this issue? Seems to be somewhat common!

**Comment by spocke** [comment]
The simple solution for this is to have a css with `p { margin: 0 }` in the editor. That is how Word and other office tools work. They have paragraphs for every "line" of text and a default margin of 0 that why it looks different. You could also set the `forced_root_block` to `div` since divs by default doesn't have any margin.

BR elements or sometimes called softbreaks are used when you press shift+enter in Word or TinyMCE so they are a different concept than paragraphs since if you format a few lines separated by soft breaks all those lines should be formatted as one single unit if you say make it a heading. So simply switching everything to soft breaks would break how rich text editors works in general.

**Reply by douglasrcjames** [reply]
Gotcha, thanks for helping clarify! Setting the paragraph tags to divs wouldnt work and would just break my current setup, and setting paragraphs to be 0 margin will make the normal paragraphs (that are not empty) look wrong as well. I may end up just forcing the editor to fix these larger line breaks on their own if copy-pasted in... going to keep investigating
